### PR TITLE
Story#99 Promisify createReadStream and createWriteStream

### PR DESCRIPTION
**Pull request by sanakanchha**
- make all the functions consistent to return a Promise
- add tests for the promisified createReadStream and createWriteStream

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +118,7 @@ options is an object or string with the following defaults:
```
Comment (brozeph) [inline README.md:126]: In the code block just above this, can you re-format as follows? (i.e. use the back tick notation "`" and qualify as Javascript) 

```javascript
{
   flags: 'r',
   encoding: null,
   fd: null,
   mode: 0x666,
   autoClose: true
 }
```

File: README.md [hunk 2]
```diff
@@ -139,7 +166,7 @@ export async function writeContent () {
```
Comment (brozeph) [inline README.md]: Same comment for the `options` object documentation above this - I can't put the comment directly on the line of code, but just prior to the `writeContent` example, there is a code block for the options. Can you put that in a block using backticks for consistency?

**Review by brozeph — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sanakanchha) [reply README.md:126]: this has been taken care of

File: README.md (hunk 2 above)
Comment (sanakanchha) [reply README.md]: taken care of

**Pull request merged**